libafl_bolts/src/tuples.rs

```diff
@@ -448,6 +456,13 @@ where
             self.1.name(index - 1)
         }
     }
+
+    fn names(&self) -> Vec<Cow<'static, str>> {
+        let first = self.0.name().to_owned();
+        let mut last = self.1.names();
+        last.insert(0, first);
```

Comment:
Let's go back to names_reversed, this code is hot/ should be as performant as possible

Reply:
@tokatoka why not?